source/crud/tests/v2/aggregate-out-readConcern.yml

```diff
@@ -0,0 +1,145 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+    - { _id: 3, x: 33 }
+
+minServerVersion: '4.1'
+
+collection_name: &collection_name 'test_aggregate_out_readconcern'
+
+tests:
+  -
+    description: "readConcern majority with out stage"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "majority" }
+        arguments:
+          pipeline:
+            - $sort: { x : 1 }
+            - $match: { _id: { $gt: 1 } }
+            - $out: &output_collection "other_test_collection"
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline:
```

Comment:
Likewise, this can also use `&pipeline` to reduce duplication in future assertions (if you end up creating an anchor for `&arguments` above).